app/controllers/posts_controller.rb

```diff
@@ -40,7 +44,7 @@ def update
     @post = Post.find(params[:id])
 
     # grab input from form
-    new_post =  params.require(:post).permit([:title, :body])
+    new_post =  params.require(:post).permit([:title, :body, {tag_ids: []}, :image])
```

Comment:
you probably dont have to save the params to a variable just pass them in the `@post.update` as an argument